dev/com.ibm.ws.ejbcontainer.bindings_fat/fat/src/com/ibm/ws/ejbcontainer/bindings/fat/tests/repeataction/RepeatOnErrorEE8.java

```diff
@@ -0,0 +1,126 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.ejbcontainer.bindings.fat.tests.repeataction;
+
+import com.ibm.websphere.simplicity.config.EJBContainerElement;
+import com.ibm.websphere.simplicity.config.ServerConfiguration;
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.custom.junit.runner.RepeatTestFilter;
+import componenttest.rules.repeater.EE8FeatureReplacementAction;
+import componenttest.rules.repeater.FeatureReplacementAction;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.impl.LibertyServerFactory;
+
+/**
+ *
+ */
+public class RepeatOnErrorEE8 extends EE8FeatureReplacementAction {
+
+    public static final String ID_EJB_ON_ERROR = EE8FeatureReplacementAction.ID + "_EjbOnErr";
+    public static final String ID_FAIL = ID_EJB_ON_ERROR + "_FAIL";
+    public static final String ID_IGNORE = ID_EJB_ON_ERROR + "_IGNORE";
+    public static final String ID_WARN = ID_EJB_ON_ERROR + "_WARN";
+
+    private static final Class<?> c = RepeatOnErrorEE8.class;
+    String serverName;
+
+    public EjbOnError onErrorAction = EjbOnError.WARN;
+
+    public RepeatOnErrorEE8(EjbOnError ejbOnError) {
+        onErrorAction = ejbOnError;
+    }
+
+    public static boolean isActive() {
+        return RepeatTestFilter.isRepeatActionActive(ID_EJB_ON_ERROR);
+    }
+
+    public static boolean isActive(EjbOnError onErrorAction) {
+        switch (onErrorAction) {
+            case FAIL:
+                return RepeatTestFilter.isRepeatActionActive(ID_FAIL);
+            case IGNORE:
+                return RepeatTestFilter.isRepeatActionActive(ID_IGNORE);
+            case WARN:
+                return RepeatTestFilter.isRepeatActionActive(ID_WARN);
+        }
+        return false;
+    }
+
+    @Override
+    public FeatureReplacementAction forServers(String... serverNames) {
+        if (serverNames == null || serverNames.length != 1) {
+            throw new IllegalStateException("Only 1 server supported by RepeatOnErrorEE9 repeat action");
```

Comment:
the exception has RepeatOnErrorEE9 printout in the RepeatOnErrorEE7 class

Reply:
corrected